install_repos

```diff
@@ -16,12 +16,9 @@ sudo apt update
 sudo apt --yes upgrade
 for repo in $(ls $TARGETDIR | grep ^$REPOPREFIX) ; do
     REQUIREMENTS=$TARGETDIR/$repo/apt-requirements.txt
-    if test -f $REQUIREMENTS ; then
+    if [ -f $REQUIREMENTS ] ; then
 	echo "Installing requirements from: "$REQUIREMENTS
-	while read line; do
-	    echo "Installing: "$line
-	    sudo apt --yes install $line
-	done < $REQUIREMENTS
+	xargs -a apt_requirements.txt -- sudo apt --yes install
```

Comment:
Was likely not clear enough in my question and explaination before.
The suggested row need to know the location of each apt_requirements.txt .
To me the full path of each apt_requirement seem to be $REQUIREMENTS ?